crates/forge/src/init.rs

```diff
@@ -0,0 +1,115 @@
+use anyhow::{bail, Context, Ok, Result};
+
+use include_dir::{include_dir, Dir};
+use scarb_metadata::MetadataCommand;
+
+use std::fs;
+use std::path::{Path, PathBuf};
+use std::process::{Command, Stdio};
+use toml_edit::{ArrayOfTables, Document, Item, Table};
+
+static TEMPLATE: Dir = include_dir!("starknet_forge_template");
+
+fn overwrite_files_from_template(
```

Comment:
```suggestion
fn overwrite_files_from_scarb_template(
```
nit, but can get lost in what template it is